os/arch/arm/src/armv7-a/arm_gicv2.c

```diff
@@ -416,7 +416,7 @@ uint32_t *arm_decodeirq(uint32_t *regs)
  *
  *   This function implements enabling of the device specified by 'irq'
  *   at the interrupt controller level if supported by the architecture
- *   (up_irq_restore() supports the global level, the device level is
+ *   (irqrestore() supports the global level, the device level is
```

Comment:
Description says `irqsave` and `irqrestore`, but real code uses `enter_critical_section` and `leave...section`. What is our plan for this?

Reply:
enter and leave critical section api are required for smp scenario. 
If CONFIG_SMP is disabled, then enter and leave critical section apis will map to irqsave and irqrestore.